pkgs/by-name/ya/yandex-cloud/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeBinaryWrapper,
+  installShellFiles,
+  buildPackages,
+  withShellCompletions ? stdenv.hostPlatform.emulatorAvailable buildPackages,
+  # update script
+  writers,
+  python3Packages,
+  nix,
+  # tests
+  testers,
+  yandex-cloud,
+}:
+let
+  pname = "yandex-cloud";
+  sources = lib.importJSON ./sources.json;
+  inherit (sources) version;
+  binaries = lib.mapAttrs (_: fetchurl) sources.binaries;
```

Comment:
What's the point of mapping here then extracting a single entry later, instead of extracting a single entry there and applying afterwards?

Reply:
Good point, I think binaries were exported via passthru in the first iterations but that is no longer the case.